Tests/LibWeb/Layout/expected/table/colspan-relaxed-parsing.txt

```diff
@@ -0,0 +1,90 @@
+Viewport <#document> at (0,0) content-size 800x600 children: not-inline
```

Comment:
typo in commit message: s/tailing/trailing/

Reply:
Dropped the commit.